### PR TITLE
Try hard to parse through broken XML

### PR DESCRIPTION
**Pull request by drnextgis**
This PR takes advantage of `recovery` option of the `lxml.etree.XMLParser` parser and aims to fix the following sort of issues:
```
>>> k = kml.KML()
>>> k.from_string(doc.encode('utf-8'))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/denis/env/lib/python3.6/site-packages/fastkml/kml.py", line 91, in from_string
    parser=etree.XMLParser(huge_tree=True)
  File "src/lxml/etree.pyx", line 3213, in lxml.etree.fromstring
  File "src/lxml/parser.pxi", line 1877, in lxml.etree._parseMemoryDocument
  File "src/lxml/parser.pxi", line 1765, in lxml.etree._parseDoc
  File "src/lxml/parser.pxi", line 1127, in lxml.etree._BaseParser._parseDoc
  File "src/lxml/parser.pxi", line 601, in lxml.etree._ParserContext._handleParseResultDoc
  File "src/lxml/parser.pxi", line 711, in lxml.etree._handleParseResult
  File "src/lxml/parser.pxi", line 640, in lxml.etree._raiseParseError
  File "<string>", line 33
lxml.etree.XMLSyntaxError: Namespace prefix lc on attachment is not defined, line 33, column 18
```

### CODE REVIEW

**Review by cleder — APPROVED** [review]

**Comment by drnextgis** [comment]
Thanks for merge! Isn't it time to release a new version?
